client/blocks/app-banner/style.scss

```diff
@@ -63,21 +69,46 @@ body.app-banner-is-visible {
 	}
 }
 
+.app-banner__icon {
+	height: 50px;
+	width: 93px;
+	margin: auto;
+}
+
 .app-banner__text-content {
 	width: 100%;
+
+	&.jetpack {
+		letter-spacing: -0.4px;
+		margin: 30px auto 0;
+	}
 }
 
 .app-banner__title {
 	/* stylelint-disable-next-line */
 	font-size: rem( 22px ); //typography-exception
 	font-family: $brand-serif;
 	line-height: 30px;
+
+	&.jetpack {
+		/* stylelint-disable-next-line */
+		font-size: rem( 28px ); //typography-exception
+		font-weight: bold;
+		padding: 0 30px;
```

Comment:
Rather than relying upon `padding`, I wonder if it might be more adaptable to set a `max-width` based upon the font size and use auto-margins. The parent element already has `padding`. Currently, the two containers combining padding means the text is quite narrow on small screens. This would apply to both the title and copy styles. WDYT? 

<details><summary>Small Screen Horizontal Spacing</summary>

| Before | After |
| - | - |
| <img width="321" alt="editor-before" src="https://user-images.githubusercontent.com/438664/187774582-1746a7de-40ae-416a-b310-f4d70e301da0.png"> | <img width="322" alt="editor-after" src="https://user-images.githubusercontent.com/438664/187774589-9248ac00-a018-4277-9669-5de4c3c463c6.png"> |
| <img width="324" alt="notifications-before" src="https://user-images.githubusercontent.com/438664/187774590-8862a7a9-0f6b-4128-83a8-ee5b9b074873.png"> | <img width="322" alt="notifications-after" src="https://user-images.githubusercontent.com/438664/187774595-9f1837f7-5e91-43a3-9d0a-f1b7f94ce0b9.png"> |

</details>

```css
.example-avoiding-long-lines {
  max-width: 18em;
  margin-left: auto;
  margin-right: auto;
} 
```

Reply:
Thank you for the thorough testing here, David, you're right that that padding looks really off on smaller screens. I've gone ahead to implement your suggested update in https://github.com/Automattic/wp-calypso/pull/67013/commits/606c625eeedfaf674b3bb1be33d4a506783b462f and I think it works much better on smaller screens now. 🙇‍♀️